runner/src/examples.rs

```diff
@@ -0,0 +1,938 @@
+        steps: vec![
+            example.construct_application_build_steps(),
+            if opt.run_server.unwrap_or(true) {
+                // Build the server first so that when running it in the next step it will start up
+                // faster.
+                vec![build_server(&opt.build_server)]
+            } else {
+                vec![]
+            },
+            if opt.build_docker {
+                vec![build_docker(&example.example)]
+            } else {
+                vec![]
+            },
+            example.construct_backend_build_steps(),
+            vec![Step::Multiple {
+                name: "run".to_string(),
+                steps: vec![run_backend_server_clients],
+            }],
+        ]
+        .into_iter()
+        .flatten()
+        .collect::<Vec<_>>(),
+    }
+}
+
+fn run_functions_example(example: &FunctionsExample) -> Step {
+    let opt = &example.options;
+
+    // build any backend server
+    let run_clients = run_clients(
+        &example.example,
+        &opt.build_client,
+        opt.client_additional_args.clone(),
+    );
+
+    #[allow(clippy::collapsible_if)]
+    let run_backend_server_clients: Step = if opt.run_server.unwrap_or(true) {
+        let run_server_clients = example.construct_example_server_run_step(run_clients);
+        example.construct_backend_run_steps(run_server_clients)
+    } else {
+        run_clients
+    };
+
+    Step::Multiple {
+        name: example.example.name.to_string(),
+        steps: vec![
+            example.construct_application_build_steps(),
+            if opt.run_server.unwrap_or(true) {
+                // Build the server first so that when running it in the next step it will start up
+                // faster.
+                vec![build_functions_server(&opt.build_server)]
+            } else {
+                vec![]
+            },
+            if opt.build_docker {
+                vec![build_docker(&example.example)]
+            } else {
+                vec![]
+            },
+            example.construct_backend_build_steps(),
+            vec![Step::Multiple {
+                name: "run".to_string(),
+                steps: vec![run_backend_server_clients],
+            }],
+        ]
+        .into_iter()
+        .flatten()
+        .collect::<Vec<_>>(),
+    }
+}
+
+fn build_wasm_module(name: &str, target: &Target, example_name: &str) -> Step {
+    match target {
+        Target::Cargo {
+            cargo_manifest,
+            additional_build_args,
+        } => {
+            let metadata = cargo_metadata::MetadataCommand::new()
+                .manifest_path(cargo_manifest)
+                .exec()
+                .unwrap();
+            Step::Single {
+                name: format!("wasm:{}:{}", name, cargo_manifest.to_string()),
+                command: Cmd::new(
+                    "cargo",
+                    // Keep this in sync with `/oak_functions/sdk/test/utils/src/lib.rs`.
```

Comment:
I think a line is missing here compared to where this was branched from.

Reply:
Thanks. Fixed.